packages/material-ui/src/Modal/ModalManager.test.js

```diff
@@ -139,6 +139,28 @@ describe('ModalManager', () => {
       assert.strictEqual(container1.style.paddingRight, paddingRightBefore);
       assert.strictEqual(fixedNode.style.paddingRight, paddingFixedRightBefore);
     });
+
+    it('should restore styles correctly if none existed before', () => {
+      // reset padding on containers
+      fixedNode.style.setProperty('padding', '');
```

Comment:
Do we need to use setProperty / removeProperty? Is it equivalent to?
```jsx
fixedNode.style.padding = x;
```

Reply:
they are equivalent except in the case of `removeProperty`

Reply:
`style.x = ''` doesn't actually seem to remove the prop, it just sets it back to whatever the previous value was, at least for hyphenated css properties